src/Context/GlobalContext/StateContext.jsx

```diff
@@ -0,0 +1,72 @@
+import { createContext, useContext, useReducer, useEffect, useState } from "react"
+import axios from "axios"
+import { uniqueCategory } from "../../utils/uniqueCategory"
+import { sortByPrice } from "../../utils/sortByPrice"
+import { stateReducerFun } from "../GlobalContext/StateReducer"
+
+const StateContext = createContext()
+
```

Comment:
maybe u can name this globalstatecontext for more specificity

Reply:
Okay